src/Ssh/Ssh/Common/SshBaseCmdlet.cs

```diff
@@ -420,6 +436,16 @@ protected internal void SetResourceType()
             {
                 IPage<GenericResource> resources = ResourceManagementClient.Resources.ListByResourceGroupWithHttpMessagesAsync(ResourceGroupName, query).GetAwaiter().GetResult().Body;
                 types = resources.Select(resource => resource.Type).ToArray();
+                foreach (var resource in resources)
```

Comment:
This list of resources is a list of all resources in the resource group with the name passed in as a parameter that is one of the supported resource types. We don't want to add the tags of all those resources, only the resource that we are actually targetting.